SpatialGDK/Source/Public/SpatialGDKSettings.h

```diff
@@ -0,0 +1,40 @@
+// // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Do we need this?

Reply:
No that is probably a rebase conflict, @Vatyx this should be safe to remove right?

Reply:
It's all commented out so would be _really_ strange if something breaks 😆

Reply:
For sure it doesn't break, but I don't know if there was intent to readd it or to remove it